### ISSUE TITLE
error about Google authentication bearer token

### ISSUE THREAD

**Issue by BeyoungLIN** [PAIR-code/lit#26]
I have got this error several times:
2020-09-01 09:58:52.889600: W tensorflow/core/platform/cloud/google_auth_provider.cc:184] All attempts to get a Google authentication bearer token failed, returning an empty token. Retrieving token from files failed with "Not found: Could not locate the credentials file.". Retrieving token from GCE failed with "Failed precondition: Error executing an HTTP request: libcurl code 6 meaning 'Couldn't resolve host name', error details: Couldn't resolve host 'metadata'".
...
2020-09-01 10:11:32.846994: E tensorflow/core/platform/cloud/curl_http_request.cc:598] The transmission  of request 0x7fef94f8e440 (URI: https://www.googleapis.com/storage/v1/b/tfds-data/o/datasets%2Fglue?fields=size%2Cgeneration%2Cupdated) has been stuck at 0 of 0 bytes for 61 seconds and will be aborted. CURL timing information: lookup time: 0.067967 (No error), connect time: 0 (No error), pre-transfer time: 0 (No error), start-transfer time: 0 (No error)
tensorflow.python.framework.errors_impl.AbortedError: All 10 retry attempts failed. The last failure: Unavailable: Error executing an HTTP request: libcurl code 42 meaning 'Operation was aborted by an application callback', error details: Callback aborted
	 when reading metadata of gs://tfds-data/datasets/glue
I found a similar closed issue but it did not work, and it seems that it may not be a problem about the version of tensorflow but GOOGLE_APPLICATION_CREDENTIALS.

**Comment by iftenney** [comment]
This looks like an issue from TFDS - can you try the fix from #22 ?

**Comment by BeyoungLIN** [comment]
yeah i saw it. i  tried changing to "try_gcs=False" but i got the same error again. Moreover, there is a new error:
TypeError: __init__() missing 2 required positional arguments: 'op' and 'message'

**Comment by iftenney** [comment]
What package versions do you have? Can you try pinning to `tensorfow=2.3.0` and `tensorflow-datasets=3.2.1` ?

**Comment by BeyoungLIN** [comment]
thanks a lot. I finally fixed it just now. It is exactly an error from the version of tensorflow-datasets. When I used `tensorflow-datasets=3.2.1` , I got the same error several times. I tried downgrading it to `tensorflow-datasets=2.1.0` and then fixed the error. And I had thought it may be a problem of the version of tensorflow, and I had tried to install `tensorflow=1.15.0`, but it did not work. Now the version I use is `tensorflow=2.3.0` and `tensorflow-datasets=2.1.0`. Thanks again.
